src/components/room-delete/roomDeleteDialogTheme.js

```diff
@@ -0,0 +1,38 @@
+import {borderDefault} from 'grape-theme/dist/web-colors'
+import {red} from 'grape-theme/dist/base-colors'
+
+export const styles = {
+  wrapper: {
+    padding: 20,
+    borderTop: {
+      width: 3,
+      style: 'solid',
+      color: borderDefault
+    },
+    maxHeight: '80vh',
+    overflowY: 'auto'
+  },
+  messageContainer: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  messageIcon: {
+    marginRight: 20,
+    height: 40,
+    fill: red
+  },
+  messageWarning: {
+    color: red
+  },
+  inputContainer: {
+    padding: [20, 0]
+  },
+  deleteButton: {
+    // FIXME once https://github.com/callemall/material-ui/issues/6227 is soved
```

Comment:
as I said MUI will install and use the local dependency `^0.9.0` even if I explicitly define `^0.10.0` in our repo